src/css/responsive/tablet.styl

```diff
@@ -60,6 +60,33 @@ svg.pagination-arrow
 
 // ******************************************************
 
+// BLOG
+
+.blog-item-single
+  max-width: none
+
+body#about
```

Comment:
theres a pretty high probability that something down the line could have #about attached to it. i dont see this as a problem